### PR TITLE
ipi-install-install: don't share the installer log

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @wking

### CODE REVIEW

File: ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh [hunk 1]
```diff
@@ -46,8 +46,7 @@ cp \
     -t "${SHARED_DIR}" \
     "${dir}/auth/kubeconfig" \
     "${dir}/metadata.json" \
-    "${dir}/terraform.tfstate" \
-    "${dir}/.openshift_install.log"
+    "${dir}/terraform.tfstate"
```
Comment (wking) [inline ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh]: Not clear to me why we need this one either.

Comment (wking) [reply ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh]: From [here][1].  Possibly a relic of SSH-based gathers, dropped in #6336.

[1]: https://github.com/openshift/release/pull/6708/commits/de3de20d02e1a596daeaeae58b9fd6858c8b5353#diff-aa102aaad663ef68f918acd81caa3cfeR48

**Comment by wking** [comment]
Added in 613473a6f (#7899), but we don't need to append to the log in destroy, we can just create a new log asset.  And the secret has a size cap, which the log file can overrun.

/lgtm

File: ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh (hunk 1 above)
Comment (bbguimaraes) [reply ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh]: It was definitely necessary when it was added.

**Pull request merged**